app/services/discovery_engine/evaluation/evaluation_resource.rb

```diff
@@ -0,0 +1,67 @@
+module DiscoveryEngine
+  module Evaluation
+    class EvaluationResource
```

Comment:
Should we think about a new name for this in this PR?

`EvaluationRunner` maybe?

Reply:
Done - but as a fresh commit.